scripts/configure_dev.sh

```diff
@@ -61,11 +89,29 @@ elif [ "${OS}" = "darwin" ]; then
     install_or_upgrade automake
     install_or_upgrade shellcheck
     install_or_upgrade python3
+elif [ "${OS}" = "windows" ]; then
+    pacman -S --disable-download-timeout --noconfirm git automake autoconf m4 libtool make mingw-w64-x86_64-gcc mingw-w64-x86_64-go mingw-w64-x86_64-boost mingw-w64-x86_64-python mingw-w64-x86_64-jq unzip procps
+    if [ $? -ne 0 ]
+    then
+        echo "Error installing pacman dependencies"
+        exit 1
+    fi
+
+    # Golang probably is not installed under MSYS2 so add the environment variable temporarily
+    export GOPATH=$HOME/go
+
+    install_windoows_shellcheck
+    if [ $? -ne 0 ]
+    then
+        exit 1
+    fi
+
+    # This is required because http://github.com/karalabe/hid library compiles with non-static libraries
+    cp /mingw64/bin/libwinpthread-1.dll $GOPATH/bin/ 
```

Comment:
I think it would be cleaner to add the `/mingw64/bin` directory to the path on compiling machines.
For packaged deployment, we need to copy the above file anyway.

Reply:
Actually is already there when you open the MSYS2 MingW64 terminal:

This is my current configuration:

```
Blaster64@BlasterNotebook MINGW64 ~
$ echo $PATH
/mingw64/bin:/usr/local/bin:/usr/bin:/bin:/c/Windows/System32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0/:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
```

Reply:
if it's already there, why do you need to copy it ? windows DLL loader would already search on the PATH, and would find it. Is there any usage of this DLL as a library source file ? i.e. building the stub out of an existing DLL ?

Reply:
Because you usually don't run the app inside MSYS2. The goal is to create executable files that can be distributed and used anywhere.

Despite it is a simple wrapper to convert pthread api into native Windows ones, I'll continue trying to remove that dependency and see if I can compile the whole code statically.

About source code, here: https://github.com/mirror/mingw-w64/tree/master/mingw-w64-libraries/winpthreads